Cargo.toml

```diff
@@ -35,6 +36,7 @@ default-members = [
     "player",
     "tests",
     "wgpu-core",
+    "wgpu-core/platform-features/*",
```

Comment:
thought: This is getting to the point that I wonder if it wouldn't be better to just have a `crates/*` directory instead. 😅